tests/container_tests/test_zip_container.py

```diff
@@ -14,21 +14,22 @@ class TestZipHandler(unittest.TestCase):
     def setUp(self):
         # The following zip layout is created for all the tests
         # outside.zip
-        # | - testdir
-        # |   | - nested.zip
-        # |   |   | - nested_dir
-        # |   |   |   | - nested
+        # | - testdir1
+        # |   | - nested1.zip
+        # |       | - nested_dir
+        # |           | - nested
+        # | - testdir2
         # |   | - testfile
```

Comment:
Test case that addresses #66 must be added. Otherwise this commit can't be a test fix.

Reply:
The tests are included at https://github.com/chainer/chainerio/blob/11b1b1a50c2abb268585871ea5dfb8ba06921397/tests/container_tests/test_zip_container.py#L119
Although they are not exactly like the examples in #66, But they can reproduce the same errors.
I will refactor the test case to make it more readable